### PR TITLE
feat(App): add setup to new CDN

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
#Checklist

- [] Check if there is a new color.
- [] Create the color variable in the _colors.scss file.
- [] Follow the color nomenclature.
- [] Create the color class with its variable.
- [] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

**Comment by andresmoschini** [comment]
@fcoronelmakingsense take into account that you are also publishing in this file: https://github.com/FromDoppler/Doppler-UI-System/blob/master/build-n-release-w-docker.sh

**Comment by andresmoschini** [comment]
Independent of these changes, you have to research what version is using all UI Library clients in order to copy them from the old CDN to the new one.

File: build-n-release-w-docker.sh [hunk 1]
```diff
@@ -80,3 +80,13 @@ docker run --rm \
...
+
+echo Publishing to our new CDN using SFTP
+echo "Destination: ${CDN_SFTP_USERNAME}@${CDN_SFTP_HOSTNAME}:${CDN_SFTP_PORT}:/${CDN_SFTP_BASE}/${pkgName}/${pkgVersion}"
+# Using specific digest (f7f7607...) to avoid unwanted changes in the non-oficial image
+docker run --rm \
+    -v /var/lib/jenkins/.ssh:/root/.ssh:ro \
+    -v "$(pwd)/dist:/source/${pkgName}/${pkgVersion}" \
+    ttionya/openssh-client@sha256:f7f7607d56f09a7c42e246e9c256ff51cf2f0802e3b2d88da6537bea516fe142 \
+    scp -P "${CDN_SFTP_PORT}" -r /source/${pkgName} "${CDN_SFTP_USERNAME}@${CDN_SFTP_HOSTNAME}:/${CDN_SFTP_BASE}/"
+echo "Files ready in https://cdn.fromdoppler.com/${pkgName}/${pkgVersion}"
```
Comment (andresmoschini) [inline build-n-release-w-docker.sh:92]: Falta subirlo a latest

File: build-n-release-w-docker.sh [hunk 2]
```diff
@@ -80,3 +80,13 @@ docker run --rm \
     -e "VERSION_NAME=latest" \
     -v `pwd`/dist:/source \
 dopplerrelay/doppler-relay-akamai-publish
+
```
Comment (cbernat) [inline build-n-release-w-docker.sh:83]: I think you are missing latest publish for new cdn.

File: build-n-publish-w-docker.sh [hunk 3]
```diff
@@ -31,3 +31,13 @@ docker run --rm \
...
     dopplerrelay/doppler-relay-akamai-publish
+
+echo Publishing to our new CDN using SFTP
+echo "Destination: ${CDN_SFTP_USERNAME}@${CDN_SFTP_HOSTNAME}:${CDN_SFTP_PORT}:/${CDN_SFTP_BASE}/${pkgName}/${pkgVersion}"
+# Using specific digest (f7f7607...) to avoid unwanted changes in the non-oficial image
+docker run --rm \
+    -v /var/lib/jenkins/.ssh:/root/.ssh:ro \
+    -v "$(pwd)/dist:/source/${pkgName}/${pkgVersion}" \
+    ttionya/openssh-client@sha256:f7f7607d56f09a7c42e246e9c256ff51cf2f0802e3b2d88da6537bea516fe142 \
+    scp -P "${CDN_SFTP_PORT}" -r /source/${pkgName} "${CDN_SFTP_USERNAME}@${CDN_SFTP_HOSTNAME}:/${CDN_SFTP_BASE}/"
```
Comment (cbernat) [inline build-n-publish-w-docker.sh:42]: Here check, I'm not so sure that source folder has a pkgName subfolder I would try:
```suggestion
    scp -P "${CDN_SFTP_PORT}" -r /source/ "${CDN_SFTP_USERNAME}@${CDN_SFTP_HOSTNAME}:/${CDN_SFTP_BASE}/${pkgName}/${pkgVersion}"
```

**Comment by cbernat** [comment]
Careful not to publish your tests :)

**Pull request merged**